util/microservice-sdk/js/examples/ConfigManager/index.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-console */
+const util = require('util');
```

Comment:
Or maybe you can put a more practical example in the context of an app... 
I don't know... Maybe it may not be as clear as how to use it in practice.

Reply:
Maybe just with the simplest case, without putting the other things that the library can do here.

Reply:
They're just ideas. I don't know if it's the best.

Reply:
It makes sense, this sample is far too simple. I already have an ideia that will contemplate all the functionalities, will implement it.